pkgs/applications/science/logic/naproche/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, haskellPackages }:
+with haskellPackages; mkDerivation {
+  pname = "Naproche-SAD";
+  version = "0.1.0.0";
+  src = fetchFromGitHub {
+    owner = "naproche-community";
+    repo = "naproche";
+    # This revision is used by Isabelle
```

Comment:
Is this strictly required or just some arbitrary pin?

Reply:
No released version is new enough, by a large margin. It seems that development was taken over by the Isabelle team, and now the closest thing to a release is the commit used by a release of Isabelle.